### PR TITLE
Story/table configuration/579

### PR DESCRIPTION
**Pull request by Kyubinhan**
* Change observations, noon forecast, and gdps data table configuration
* Create a reusable sortable table component
* Add & modify Cypress tests so that they pass in Firefox
* Modify getTickValues behavior a little bit to limit the number of tick values

### CODE REVIEW

File: web/src/features/fireWeather/components/SortableTableByDatetime.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+})
+
+interface WeatherValue {
+  datetime: string
+  temperature?: number | null
+  relative_humidity?: number | null
+  wind_direction?: number | null
+  wind_speed?: number | null
+  precipitation?: number | null
+  total_precipitation?: number | null
```
Comment (andrea-williams) [inline web/src/features/fireWeather/components/SortableTableByDatetime.tsx:37]: when is `precipitation` used and when is `total_precipitation` used?

File: web/src/features/fireWeather/components/NoonWeatherValueTable.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    align: 'right',
+    format: (value: number) => value.toFixed(MODEL_VALUE_DECIMAL)
+  },
+  {
+    id: 'total_precipitation',
+    label: 'Precip (mm/cm)',
+    minWidth: 70,
+    maxWidth: 100,
+    align: 'right',
+    format: (value: number) => value.toFixed(MODEL_VALUE_DECIMAL)
```
Comment (andrea-williams) [inline web/src/features/fireWeather/components/NoonWeatherValueTable.tsx:62]: the precip values aren't being displayed in the noon GDPS values table in your PR's route

**Review by andrea-williams — CHANGES_REQUESTED** [review]
Global noon values table isn't displaying precip values

File: web/src/features/fireWeather/components/NoonWeatherValueTable.tsx (hunk 2 above)
Comment (Kyubinhan) [reply web/src/features/fireWeather/components/NoonWeatherValueTable.tsx:62]: Ah shoot we don't use total_precipitation property for models anymore

File: web/src/features/fireWeather/components/SortableTableByDatetime.tsx (hunk 1 above)
Comment (Kyubinhan) [reply web/src/features/fireWeather/components/SortableTableByDatetime.tsx:37]: `precipitation` is for observation, `total_precipitation` is for forecast, and there will be another one for `delta_precipitation` for model

**Review by andrea-williams — APPROVED** [review]
👍

**Review by Sybrand — APPROVED** [review]

**Review by Sybrand — CHANGES_REQUESTED** [review]
Can we address the sonarcloud issue? "Failed condition 20.1% 20.1% Duplicated Lines (%) on New Code (is greater than 3%)"

**Review by Sybrand — APPROVED** [review]

**Pull request merged**